tools-edge/oas-checker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:14.11.0-alpine3.12
+
+RUN apk update && \
+    apk upgrade && \
+    npm install -g npm install -g @stoplight/spectral
+
+# Load default ruleset from a suitable source
+RUN wget https://teamdigitale.github.io/api-oas-checker/spectral.yml -qO .spectral.yml
+
```

Comment:
It would be great if the rules file was part of the tool directory instead of downloading every time the image is built so people can see what are the rules they're using with the version of the tool.

Docker images are generated every time they're modified.

Reply:
Please include the rules file in the tool directory and copy it inside the container.